simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderActivity.java

```diff
@@ -24,15 +24,15 @@
 import android.widget.ProgressBar;
 import android.widget.TextView;
 
-import com.io7m.jfunctional.FunctionType;
 import com.io7m.jfunctional.Option;
 import com.io7m.jfunctional.OptionType;
+import com.io7m.jnull.NonNull;
```

Comment:
non-blocking/non-pertinate:

It looks like this `jnull` library is unmaintained, perhaps we should consider removing/replacing it.

https://github.com/io7m/jnull/issues/1

Reply:
The other package referenced above `jfunctional` also appears to be unmaintained:

https://github.com/io7m/jfunctional